assets/js/voidsearch.js

```diff
@@ -182,6 +200,13 @@
         );
     }
 
+    function getUrlParameter(name) {
```

Comment:
aw, is there no standard function to do this? :/

Reply:
Note that I don't actually know any JS.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams

Reply:
There is a standard function that has support for all browsers but IE. Do we care about IE? If not, I can switch over to the built-in.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
I think it's fair to not care about IE here.

Reply:
Ok, I'll swap it out.